### ISSUE TITLE
Change Tabs does not work - also when changing the keybindung

### ISSUE THREAD

**Issue by thopaw** [jesseduffield/lazygit#2839]
**Describe the bug**
Changing tabs within a oanel is not working. I installed lazygit with the provided binary on my linux laptop

**To Reproduce**

1. Open lazygit
2. press "3" for changing to the branches panel
3. press ] to switch to "remotes" or press it twice to switch to "tags"
4. the tab is **not** changed

**Expected behavior**
After pressing ] the tab should change to the next one

**Screenshots**

![image](https://github.com/jesseduffield/lazygit/assets/6630597/ccdb58fd-0675-44b8-b37f-7561a6b58322)

**Version info:**

lazygit version

> commit=35c430c8c9a7eb35d79897fc04a4c444dfd95321, build date=2023-07-24T11:19:14Z, build source=binaryRelease, version=0.39.4, os=linux, arch=amd64, git version=2.34.1

git version

> git version 2.34.1

**Additional context**
Also when changing the bindings to other keys, changing the tabs is not working.

```
keybinding:
  universal:
    increaseContextInDiffView: ')'
    decreaseContextInDiffView: '('
    nextTab: ')'
    prevTab: '('
```
I needed to change the keys, as in version 0.35 the keys "[" and "]" are not working (same as in the new version)

I am using a german keyboard where [ and ] are typed by "Alt Gr" + 8 or 9.
I guessed that this was the issue why it worked for me when switching to round brackets in version 0.35.

**Comment by mark2185** [comment]
Could you please try building the latest `master`? This should have fixed such issues with non-US layouts - https://github.com/jesseduffield/lazygit/pull/2818

**Comment by thopaw** [comment]
I cloned the repo and called go build

With the created binary the problem is the same.

```
[I] ╭─thopaw at thopaw-XPS-13-9310 in /home/thopaw/projects/lazygit (master ✔)
╰─λ ./lazygit --version                                                                                                                                                                                   0 (0.223s)
commit=6be9109aaa5dd0f224c06bca23e00bc89b5c5c2e, build date=2023-07-26T12:34:58Z, build source=unknown, version=6be9109a, os=linux, arch=amd64, git version=2.34.1
[I] ╭─thopaw at thopaw-XPS-13-9310 in /home/thopaw/projects/lazygit (master ✔)
╰─λ                   
```

**Comment by thopaw** [comment]
Changing the tabs with the mouse is working.
So it "only" a keybinding issue

**Comment by pcorbineau** [comment]
Still got the issue with version 0.40.2, on Windows (WinGet version) with an AZERTY keyboard (French Layout)

**Comment by joeldegerman99** [comment]
Still an issue with the keybindings

**Comment by danielwikander** [comment]
I am also experiencing this. 
(On Windows, Swedish layout)

**Comment by MarcoBuess** [comment]
Same issue 🙌 . Windows, German Layout

**Comment by stefanhaller** [comment]
> This should have fixed such issues with non-US layouts - #2818

That fix only fixed it for characters that you type into an editor (e.g. prompt); it didn't affect keybindings.

Actually I think that fix was wrong, the problem should be fixed at a different level. For those who experience the issue and can build from source, could you please try applying the following patch to see if that fixes it, and doesn't cause other problems? If this works, we'll have to make a PR for tcell (and then probably revert #2818).

```diff
diff --git a/vendor/github.com/stefanhaller/tcell/v2/console_win.go b/vendor/github.com/stefanhaller/tcell/v2/console_win.go
index 01fa1f7b..174f3ae9 100644
--- a/vendor/github.com/stefanhaller/tcell/v2/console_win.go
+++ b/vendor/github.com/stefanhaller/tcell/v2/console_win.go
@@ -647,8 +647,8 @@ func mod2mask(cks uint32) ModMask {
 	if (cks & (0x0008 | 0x0004)) != 0 {
 		mm |= ModCtrl
 	}
-	// Left or right alt
-	if (cks & (0x0002 | 0x0001)) != 0 {
+	// Left or right alt; set this only if ctrl is not also set, since ctrl+alt means AltGr
+	if (cks & (0x0002 | 0x0001)) != 0 && (cks & (0x0008 | 0x0004)) == 0 {
 		mm |= ModAlt
 	}
 	// Any shift
```

I really need help testing this. I can only do limited testing, because I access my Windows machine through RDP, and who knows how many translation layers the keyboard handling goes through in that case.

**Comment by pcorbineau** [comment]
@stefanhaller  
I confirm this patch make it works using AZERTY config on Windows.
'['  key is now working ( ALTGR + 5).

I didn't test it too much so I don't know if there is side effects.

**Comment by stefanhaller** [comment]
Here's a PR that should fix this: #3194. I'd appreciate it if people affected by the problem could test it.

**Comment by stefanhaller** [comment]
Should be fixed by #3194.
